### ISSUE TITLE
CGX graph enhancement

### ISSUE THREAD

**Issue by 3rav** [mkraska/CalculiX-Examples#23]
Added the ability to create plots datasets vs. datasets (STRES vs. STARIN):

graph set vs STRES SXX STRAIN EXX

**Comment by mkraska** [comment]
Yes, that would be convenient. Currently, my preferred workaround is to write the data to the selected results file (DAT) and use the helper script dat2txt to generate text files with space separated columns. 

For use with gnuplot you have to join the columns of individual output quantities using the system command join. Maybe there is a direct way in gnuplot to plot columns from different files....
